### ISSUE TITLE
Unknown Field Found (i18n_lang and i18n_refs)

### ISSUE THREAD

**Issue by nicktaylor** [sanity-io/document-internationalization#89]
We were on the old internationalization plugin and in order to upgrade to Studio V3 needed to migrate over to this version of the plugin. We have field names of `i18n_lang`, `i18n_refs` and `i18n_base`, which are set and passed over as a parameter of `withDocumentI18nPlugin` to ensure they are correctly mapped. We have also applied `i18n: true` on the applicable documents.

Firstly, everything works great, and works against all our original data and i18n references. However, we're seeing the following message at the bottom of all our i18n docs:

Unknown field found
Encountered a field that is not defined in the schema.

This appears for both the `i18n_lang` and `i18n_refs` fields. How can these be hidden / ignored from the Studio? Is this a bug?

Sanity Studio Version: 3.0.6
Document Internationalization Version: 1.0.2

**Comment by alex-ketch** [comment]
I don't think the documentation has been fully updated yet, but try looking at the [docs in the `studio-v3-plugin-v2` branch](https://github.com/sanity-io/document-internationalization/tree/studio-v3-plugin-v2#usage)

**Comment by nicktaylor** [comment]
Thanks @alex-ketch , however, that appears to be for the currently unreleased v2.0 plugin. I'd prefer not to move over to that yet.

This appears to be a bug where there are custom field names, rather than using the ones with double underscore at the start. If I switch to the default fields, I don't get the 'Unknown field found' errors.

I found the schema that used to need to be added in order to get GraphQL working:
https://github.com/sanity-io/document-internationalization/blob/0b22ea2fa1f018c04f0c175d3fdc51ec0b116395/docs/graphql-intl-doc.md

We're not using GraphQL, however, I've added this to every document that's using translations and that's hidden the errors for the time being.

TL;DR: This appears to be a bug with custom field names.

**Comment by LiamMartens** [comment]
@nicktaylor the only way to handle this is to actually add those field to the schemas that need it and subsequently set them to hidden in the schema. This is due to the way Sanity handles it's schemas
